SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -581,6 +581,12 @@ public SingularityRequestParent scale(@ApiParam("The Request ID to scale") @Path
     validator.checkScale(newRequest, Optional.<Integer>absent());
 
     checkBadRequest(oldRequest.getInstancesSafe() != newRequest.getInstancesSafe(), "Scale request has no affect on the # of instances (%s)", newRequest.getInstancesSafe());
+    String scaleMessage = String.format("Scaling from %d instance(s) to %d instances", oldRequest.getInstancesSafe(), newRequest.getInstancesSafe());
```

Comment:
given the size of the area it is displayed in the ui, this can probably be even shorter. i.e. `from %d to %s` or even `%d -> %d`